index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Conference</title>
+  <meta name="description" content="Conference Page">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="stylesheets/css/styles.css">
+</head>
+
+<body>
+  <header id="toolbar">
+    <div id="top-bar">
+      <ul class="menu">
+        <li><a class="nav-link" href="#about-me"> <img src="assets/images/facebook.png" alt=""> </a></li>
+        <li><a class="nav-link" href="#about-me"> <img src="assets/images/twitter.png" alt=""> </a></li>
+        <li><a class="nav-link" href="#contact-form">English</a></li>
+      </ul>
+    </div>
+    <div id="bottom-bar">
+      <div id="first-item">
+        <span id="logo-placeholder"><a class="nav-link" href="index.html#intro"><img src="assets/images/Lome_Fintech_LogoB_59.png" alt=""></a></span>
+        <nav id="nav-bar">
+          <ul class="menu">
+            <li><a class="nav-link" href="about.html#about">About</a></li>
+            <li><a class="nav-link" href="index.html#main-program">Program</a></li>
+            <li><a class="nav-link" href="index.html">Join</a></li>
+            <li><a class="nav-link" href="index.html#partner">Sponsor</a></li>
+            <li><a class="nav-link" href="index.html#news">News</a></li>
+            <li><a class="nav-link" href="index.html#partner"><button class="btn">The Challenge</button></a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- Hamburger icon -->
+      <label id="second-item" class='menu-button-container' for="menu-button-container">
+        <img id="cancel-icon" src="assets/images/close_mobile_menu.png" alt="Cancel Menu">
+        <img id="menu-icon" src="assets/images/mobile_menu.png" alt="Cancel Menu">
+      </label>
+    </div>
+  </header>
+  <!-- Main content -->
+  <main class="content">
+    <section id="intro">
+      <!--       <h1 id="hello">"Hello. Sharing world"</h1>
+      <p id="title">
+        CONFERENCE INTERNATIONALE DES EXPERTS REGIONAUX EN CYBERSECURITE
+      </p> -->
```

Comment:
Kindly remove commented pieces of code in order to make the code base more concise and professional.